hw/syn/data/dc.hjson

```diff
@@ -0,0 +1,20 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  tool_srcs: ["{proj_root}/hw/foundry/syn/{tool}/setup.tcl"
+              "{proj_root}/hw/syn/tools/{tool}/run-syn.tcl"
+              "{proj_root}/hw/syn/tools/{tool}/constraints.sdc"]
+
+  exports: [{"TOP_ENTITY": "{dut}"}]
```

Comment:
Can this just be $dut?

Reply:
do you mean we should omit the quotes around `{dut}` it?

Reply:
if your question was referring to the actual name of the top-level: yes, since we do not instantiate the top-level with additional parameters which would then get appended to the name in DC (after elab).

Reply:
Ohh, I mean rename 'TOP_ENTITY' to 'dut', considering you are already registering "dut" as a key in the hjson. Having different keys that represent that same thing makes it slightly hard to understand.

Reply:
ah got it :). sure, I'll have to change some env variables anyway to be able to run the synthesis scripts from a different folder than the `sv_flist_gen_dir`...

Reply:
ok I changed this env variable to DUT now.